source/contentRecog/recogUi.py

```diff
@@ -162,17 +166,30 @@ def _onFirstResult(self, result: Union[RecognitionResult, Exception]):
 	def _scheduleRecognize(self):
 		core.callLater(self.recognizer.autoRefreshInterval, self._recognize, self._onResult)
 
+	@script(
+		# Translators: Describes a command.
+		description=_("Refresh the recognition result"),
+		# gesture="kb:NVDA+f5",
```

Comment:
should this be commented?

Reply:
No, it was not meant to be commented. This comment was an oversight after having tested if the script was inheritating something else.

It is now fixed in fb76953a26b66de19e1ae26e04ab15e4752bbb96.